app/components/UI/AccountSelectorList/AccountSelector.test.tsx

```diff
@@ -197,4 +187,13 @@ describe('AccountSelectorList', () => {
       expect(toJSON()).toMatchSnapshot();
     });
   });
+  it('should render correct account names', async () => {
```

Comment:
nit

```suggestion
  it('renders correct account names', async () => {
```

According to our [guidelines](https://github.com/MetaMask/contributor-docs/blob/main/docs/e2e-testing.md#guidelines)